js/build_map.js

```diff
@@ -103,11 +103,12 @@ function create_map() {
   // add watermark
   addWatermark();
   
-  // add placeholder group for county boundaries
-  map.append('g').classed("county-bounds", true);
+  // add placeholder groups for state and county boundaries
+  map.append('g').attr('id', 'state-bounds');
```

Comment:
I think this is the difference for why we can't see foreground states as bold. I added counties underneath the state bounds

Reply:
Just tested on your branch that I pulled down. If you just switch these lines so that the county group is added first, then it should work!